extensions/deploy_replace_undeploy/standard/sections/clause_6_deploy_replace_undeploy.adoc

```diff
@@ -151,6 +151,8 @@ include::../requirements/deploy-replace-undeploy/deploy/REQ_response-body.adoc[]
 
 See <<deploy-replace-undeploy-http_status_codes>> for general guidance.
 
+include::../requirements/deploy-replace-undeploy/deploy/REQ_unsupported-media-type.adoc[]
```

Comment:
The file seems to be named "REQ_unsupported-content-type.adoc" instead.

Reply:
I decided to rename the file to unsupported-media-type.

See the fix in c6a84f7.